sample-code/apps/TestApp/README.md

```diff
@@ -0,0 +1,11 @@
+## Repository
+- https://github.com/appium/ios-test-app
```

Comment:
Because https://github.com/appium/ios-test-app is the repository to build `TestApp`.